### PR TITLE
bumping compatible mlflow versions

### PR DESCRIPTION
**Pull request by imrehg**
Signed-off-by: Gergely Imreh <gergely.imreh@faculty.ai>

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py34, py35, py36, py37, py38, flake8, black, license
+envlist = py27 py34, py35, py36, py37, py38, flake8, black, license
```
Comment (acroz) [inline tox.ini]: Lost a comma here. Could also use tox's expansion syntax and drop Python 3.4 testing (which is no longer supported):
```suggestion
envlist = py{27,35,36,37,38}, flake8, black, license
```
(Note, I realise by my logic Python 2.7 support should be dropped, however we still need it to work until we drop Python 2.7 from the platform.)

**Review by acroz — COMMENTED** [review]
Looks good - I've released your changes to `mlflow-faculty` as 0.5.0, so could bump and test this here now.

File: tox.ini (hunk 1 above)
Comment (imrehg) [reply tox.ini]: Good point, was hesitating on doing that more concise setup, not entirely sure why, maybe readability. But in the end this feels indeed clearer, added!

**Comment by acroz** [comment]
Is this tested on platform?

**Comment by imrehg** [comment]
@acroz yeah, tested loading models saved by the latest `mlflow-models` lib, sklearn that worked before, and als loading an `xgboost` model that is newly supported after the version bump. Both seem to work fine.

**Review by acroz — APPROVED** [review]

**Comment by acroz** [comment]
Thanks for this Gergely! Released as 0.2.0.

**Pull request merged**